### ISSUE TITLE
Archive carpentry

### ISSUE THREAD

**Issue by jezcope** [data-lessons/librarycarpentry#24]
Details and discussion at LibraryCarpentry/librarycarpentry.github.io#25

**Comment by jezcope** [comment]
Ideally, the list of workshops should be:

1. Rendered on the website; and
2. Downloadable in machine-readable form e.g. JSON

@weaverbel Not sure if we need to do this ourselves or just tap into AMY?

**Comment by drjwbaker** [comment]
Working on this at the British Library site. First task is to review the call for feedback from archivists.

**Comment by drjwbaker** [comment]
Quick update: lots of progress. People working on alt data intro for librarians. Lots of findings to report back tomorrow!

**Comment by JennyBunn** [comment]
Was working on this yesterday with Charlotte and Sanayah, but can't make it to London today, so will do a quick update here. We reviewed the results of the survey and came up with a cluster of responses that were essentially talking about data migration and manipulation, import/export, cleansing and that sort of thing. This cluster included talk of XML/EAD/EAC, .csv and Excel, Python, OpenRefine, aggregators such as Archives Hub, VIAF and Trove. Also on the fringes, APIs were mentioned. Then there were a few outliers such as one mention of web scraping and one of visualisation.
Given the data focus, we thought that it made sense to review the Data Intro for Librarians lesson to see if it could be made to work as Data Intro for Archivists. We figured it might need a few changes and agreed to work on that as a first step. Charlotte and I are working on it, but I for one will probably work outside GitHub for now as it is still a bit of a mystery to me. Will upload further thoughts here though.

**Comment by charlottekostelic** [comment]
@jennybunn is working on editing the Jargon Busting episode of Data Intro for Archivists and I am working on the Foundations section. We are editing the focus of the lesson so that we look at archival collections and archival description as data. This can still allow for an introduction to computational approaches and the tool that will be taught in the last episode will still be Regular Expressions.

**Comment by drjwbaker** [comment]
I've made https://github.com/data-lessons/data-intro-archives (thanks to @weaverbel for the help). I've made @charlottekostelic an admin. I suggest this goes in the incubator cc @jezcope. I suspect we'll work more on it post sprint: for example, adding to both Lib and Arch intro lessons something in the Instructor Notes about using material from both lessons if you want. But this is start, and I know both @charlottekostelic and @JennyBunn have material that will be available soon!
